### PR TITLE
Update inv-transform.R (unif_binomial function)

### PR DESCRIPTION
**Pull request by rohitrk06**
Support for binom(n,p) is 0,1,2,...,n. Previous code does not count for this. 

Also, integer returned by cut function will not include 0, that's why subtracting 1 to shift the output by 1.

their might be some function that perform the following task without this way around. But i am new to R

### CODE REVIEW

File: w1-pseudorandom-number-generation/inv-transform.R [hunk 1]
```diff
@@ -9,12 +9,14 @@ unif_bernoulli <- function(n_samples, p) {
 
 unif_binomial <- function(n_samples, n, p) {
   x <- runif(n_samples)
-  cdf <- unique(pbinom(1:n, n, p))
-  return(cut(x, cdf, labels=FALSE))
+  #cdf <- unique(pbinom(1:n, n, p))
+  #return(cut(x, cdf, labels=FALSE))
```
Comment (jaidevd) [inline w1-pseudorandom-number-generation/inv-transform.R]: Please remove the above two commented lines.

File: w1-pseudorandom-number-generation/inv-transform.R [hunk 2]
```diff
@@ -9,12 +9,14 @@ unif_bernoulli <- function(n_samples, p) {
...
+  cdf <- unique(pbinom(-1:n, n, p))
+  return(cut(x, cdf, labels=FALSE)-1)
```
Comment (jaidevd) [inline w1-pseudorandom-number-generation/inv-transform.R]: Please put spaces around the `-` sign here.

**Review by jaidevd — CHANGES_REQUESTED** [review]
Thanks a lot for this @rohitrk06, this is perfect!

We need a CDF that contains 0 as the first element and 1 as the last element. I realized this mistake in the graded assignment but forgot to make the correction here.

The PR is fine apart from a couple of minor changes. Please make the requested changes and we'll merge this.

Thanks again,

**Comment by rohitrk06** [comment]
@jaidevd, I have made required changes suggested by you. Please have a look at them

**Comment by jaidevd** [comment]
Thanks, @rohitrk06 - keep it coming.

**Pull request merged**